dspace/solr/search/conf/schema.xml

```diff
@@ -194,11 +197,17 @@
             <!--Treats the entire field as a single token, regardless of its content-->
             <tokenizer class="solr.KeywordTokenizerFactory"/>
 
-            <!--<filter class="solr.LowerCaseFilterFactory" />-->
+            <filter class="solr.LowerCaseFilterFactory" />
```

Comment:
why

Reply:
I copied set up from the v5 (it is OK's update)